### DISCUSSION TITLE
"Skip to content" doesn't work

### DISCUSSION THREAD

**Discussion by cihad** [community/community#54489] [Accessibility]
### Select Topic Area

Product Feedback

### Body

"Skip to content" link on github.com seems doesn't work. Thats all.

**Comment by ghostinhershell** [comment]
Hi @cihad 👋, welcome to the Community! We're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by cehfisher** [comment]
Hi @cihad - this missing functionality appears to be resolved now. Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)

**Comment by cihad** [comment]
Yes, it works like a charm. :)
